### ISSUE TITLE
Visually-hidden class might merge words for screen readers

### ISSUE THREAD

**Issue by selfthinker** [alphagov/govuk-frontend#1032]
The 1px width in the (govuk-)visually-hidden class can cause screen readers to merge all hidden words together if used together with `word-wrap: break-word`.
You can see it in action on https://insidegovuk.blog.gov.uk/ where the hidden "Posted on" before a date gets read out as "Postedon" by any screen reader used with Firefox. (Check out what the "text leaf" says in the Accessibility Inspector to see it clearly.)

Although that is a [bug in Firefox](https://bugzilla.mozilla.org/show_bug.cgi?id=1494746) (which I reported last month), I wonder if we can just remove the `width: 1px`.
That depends on why the 1px width is there. A [WebAIM article](https://webaim.org/techniques/css/invisiblecontent/) says:
> While this is a likely a little overkill and can probably be omitted in most circumstances, there are a few instances where positioning may be disabled, but all other styles remain enabled.

I guess it depends on which is more likely, visually hidden content with positioning disabled or visually hidden content used in a "break-word" context.

**Comment by nickcolley** [comment]
GOV.UK Frontend
```css
.govuk-visually-hidden {
  position: absolute;
  width: 1px;
  height: 1px;
  margin: -1px;
  padding: 0;
  overflow: hidden;
  clip: rect(0 0 0 0);
  -webkit-clip-path: inset(50%);
          clip-path: inset(50%);
  border: 0;
  white-space: nowrap;
}
```

GOV.UK Blog:
```css
.visually-hidden, .visuallyhidden {
    position: absolute;
    left: -9999em;
    top: auto;
    width: 1px;
    height: 1px;
    overflow: hidden;
}
```

**Comment by 36degrees** [comment]
For what it's worth, I tried editing the blog stylesheet and replacing the visually-hidden rules with those from GOV.UK Frontend, and the leaf text in the accessibility inspector then displayed with the expected whitespace.

I also created an example in the review app using `word-wrap: break-word` and the visually hidden class and, again, the text looks correct in the accessibility inspector.

```
<p class="govuk-body" style="word-wrap: break-word;">
  This content includes some visually hidden text which will only be read by screen readers.
  <span class="govuk-visually-hidden">This is the visually hidden text, which will only be read by screen readers</span>
</p>
```

<img width="1392" alt="screen shot 2018-10-16 at 10 28 21" src="https://user-images.githubusercontent.com/121939/47006616-3b655e80-d12e-11e8-9efa-475d0c2abd5a.png">

The `white-space: nowrap` in the visually hidden class appears to be what corrects it?

I haven't tested with any assistive tech yet, but based on that I'm not sure that this issue exists in GOV.UK Frontend?
